tests/bwc/test_upgrade.py

```diff
@@ -177,24 +177,26 @@ def _test_upgrade_path(self, versions: Tuple[VersionDef], nodes):
             self._do_upgrade(cluster, nodes, paths, versions)
         except Exception as e:
             msg = ""
-            cluster_name = cluster.nodes()[0].cluster_name
-            logs_path = cluster.nodes()[0].logs_path
-
             msg = "\nLogs\n"
-            msg += "--------------\n"
-            logfile = os.path.join(logs_path, cluster_name + ".log")
-            with open(logfile, "r") as f:
-                logs = f.read()
-                msg += logs
-            msg += "\n"
+            msg += "==============\n"
+            for i, node in enumerate(cluster.nodes()):
+                msg += f"-------------- {i}\n"
```

Comment:
minor: for better clarity
```suggestion
                msg += f"-------------- node: {i}\n"
```